### ISSUE TITLE
loss plot during pretraining

### ISSUE THREAD

**Issue by davoudisaeedeh** [nadavbra/protein_bert#79]
Hi,

I'd like clarification regarding the loss plot presented in the paper. I noticed a pattern where, whenever the lines for 512 and 1024 ascend, the line corresponding to 128 descends, and vice versa for both annotation and sequence loss plots. Could you provide insights into the rationale behind this behavior?

![image](https://github.com/nadavbra/protein_bert/assets/99234583/6a8248da-cbe7-43a9-b8aa-226600f415eb)

Thanks,

**Comment by nadavbra** [comment]
If I remember correctly it's because the pretraining is split into epochs/stages with different lengths, alternating between 128, 512 and 1024, but never training of these sequences at exactly the same time.
